src/Microsoft.Health.Fhir.CosmosDb.Core/Features/Storage/StoreProcedures/IStoredProcedureMetadata.cs

```diff
@@ -3,11 +3,16 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
+using System.Collections.Generic;
```

Comment:
I believe we can clean the using statements in this file. Probably we can make it simpler.

Reply:
done